driver/src/price_estimation/dexag/api.rs

```diff
@@ -72,25 +66,23 @@ impl DexagApi for DexagHttpApi {
             .context("failed to parse token list json from dexag response")
     }
 
-    fn get_price(&self, from: &Token, to: &Token, amount: EitherAmount) -> Result<Price> {
+    fn get_price(&self, from: &Token, to: &Token) -> Result<f64> {
         let mut url = self.base_url.clone();
         url.set_path("price");
         {
             // This is in its own block because we are supposed to drop `q`.
             let mut q = url.query_pairs_mut();
             q.append_pair("from", &from.symbol);
             q.append_pair("to", &to.symbol);
-            match amount {
-                EitherAmount::From(amount) => q.append_pair("fromAmount", &amount.to_string()),
-                EitherAmount::To(amount) => q.append_pair("toAmount", &amount.to_string()),
-            };
+            q.append_pair("fromAmount", "1");
```

Comment:
This means that we will get the prices for a sellAmount of 1 DAI, right? I think that is an excellent solution to the slippage challenge with this price feed.

Reply:
If this is called with `from == DAI` and `to == ETH` then this will give the price of 1 DAI in ETH which is something like 0.004. Writing this I realize that the comment I put on this function at the top in the trait is wrong.

Reply:
> If this is called with from == DAI and to == ETH then this will give the price of 1 DAI in ETH which is something like 0.004. 

This means that there is some minimal slippage right? If I would sell 0.001 DAI, I would maybe get a better price.
While it is not important, I see that the `fromAmount` is optional. Maybe we should just leave it out and it will give us a price "without slippage"